src/Pages/Home.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { useData } from "../Context/DataProvider";
+import {
+  CheckIfVideoExistsInList,
+  formatDate,
+  formatNumber,
+  formatString
+} from "../utils";
+import { Avatar } from "../Components/Avatar";
+import { Link } from "react-router-dom";
+import { HiDotsVertical } from "react-icons/hi";
+import { MdWatchLater, MdPlaylistAdd, MdRemoveCircle } from "react-icons/md";
+
+export function Home() {
+  const {
+    state: { videos, watchLater, searchedItems },
+    addToHistory,
+    addToWatchLater,
+    handlePlaylist
+  } = useData();
+  function getfilteredData() {
```

Comment:
Please rename this function name as per camelcase.

Reply:
yes renamed it to getFilteredData()

Reply:
Great